src/views/SignUp.vue

```diff
@@ -0,0 +1,470 @@
+            {{snack2}}
+             <v-btn dark text @click="snackbarError=false">Close</v-btn>
+        </v-snackbar>
+        <v-snackbar v-model="snackbarPassword" top:timeout="timeout" color="error">
+            {{snack3}}
+             <v-btn dark text @click="snackbarPassword=false">Close</v-btn>
+        </v-snackbar>
+    </v-container>
+</template>
+
+
+<script lang="ts">
+
+import Vue from "vue";
+import Component from "vue-class-component";
+import {Watch} from "vue-property-decorator";
+import Translate from "../components/Translate.vue";
+
+@Component({ components: { Translate } })
+export default class SignUp extends Vue {
+    $store: any;
+    $router: any;
+
+    user: {identification: string, firstName: string, secondName: string,
+    lastName:string, secondLastName: string, birthday:string, email:string, password: string
+    phoneNumber:string, charge: string, idLanguage: number, idStatus: number} = {
+                identification: "",
+                firstName: "",
+                secondName: "",
+                lastName: "",
+                secondLastName: "",
+                birthday: "",
+                email: "",
+                password: "",
+                phoneNumber: "",
+                charge:  "Client",
+                idLanguage: 1,
+                idStatus: 4,
+            };
+                
+    
+    //////Variables estaticas/////
+
+                identification= "Identitificacion"
+                firstName= "First name"
+                secondName= "Second name"
+                secondNameP= "Second name (Optional)"
+                lastName= "Last name"
+                secondLastName= "Second last name"
+                secondLastNameP= "Second last name (Optional)"
+                birthday= "Birthday"
+                email= "E-mail"
+                password= "Password"
+                passwordc= "Confirm password"
+                phoneNumber= "Phone number"
+                languageInput= "Language"
+                goBack=  "Go back"
+                buttonSignup= "Sign up"
+                termCondition= "Accept the terms and conditions"
+                titlePage= "Sign up"
+                dateHint="MM/DD/YYYY format"
+                snack1 = "User registered successfully"
+                snack2 = "User registration error. Try again."
+                snack3 = "Please confirm password correctly"
+
+    //////Fin variables estaticas//////
+
+    password2= "";
+    items: Array<string> = ['English', 'Spanish'];
+    nameLanguage = "";
+    snackbar = false;
+    snackbarError = false;
+    snackbarPassword = false;
+    timeout =7000;
+
+   
+
+    searchRoute() {
+      if (this.nameLanguage=="English"){
+      this.user.idLanguage = 1;
+    } else if (this.nameLanguage == "Spanish") {
+      this.user.idLanguage = 2;
+    }
+        if (this.$refs.form.validate()&&this.password2===this.user.password){
+             console.log(this.user);
+            this.$store
```

Comment:
Cambiar por:
this.$store
        .dispatch("user/createUserRoute", this.user)
        .then(() => {
          this.snackbar = true;
          setTimeout(() => {
            this.changePage("Home");
          }, 1000);
        })
        .catch(() => {
          this.snackbarError = true;
        });